mkdir: cannot create directory ‘/tmp/tobiastmp/’: File exists
Traceback (most recent call last):
  File "./preprocess5.py", line 4, in <module>
    from handle_sequences22 import read_fastq, read_fastq_paired_end
ModuleNotFoundError: No module named 'handle_sequences22'

real	0m0.181s
user	0m0.041s
sys	0m0.013s
cp: cannot stat ‘/tmp/tobiastmp/outtest5’: No such file or directory
mkdir: cannot create directory ‘/tmp/tobiastmp/’: File exists
2018-11-22 09:26:10 Starting UMI Error Correct
2018-11-22 09:26:10 Writing output files to /tmp/tobiastmp/outtest5

real	101m38.693s
user	101m32.697s
sys	0m5.539s
